modules/infra/control/control_output.c

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+
+#include <gr_control.h>
+#include <gr_control_output.h>
+#include <gr_graph.h>
+#include <gr_log.h>
+#include <gr_macro.h>
+#include <gr_mbuf.h>
+
+#include <event2/event.h>
```

Comment:
Move this include with non dpdk incluses.

Reply:
the header sorting is done by clang-format:

https://github.com/DPDK/grout/blob/2d90e7ccdd257a5155a47a62ea9304910d6b7038/.clang-format#L89-L100

Reply:
it depends on clang version though. (I applied clang format, it didn't fixed it).
Same as '-Wmissing-variable-declarations' which isn't supported in my system. (RHEL 9.4, gcc version 11.4.1 ...)

Reply:
Your formatting correctly follows what is configured for clang-format. I was replying to David's remark :)